add support for headless message signing (#13)
meso-network/meso-js: packages/meso-js/README.md

  network: Network; // The network to use for the transfer
  walletAddress: string; // The user's wallet address obtained at runtime by your application
  layout?: Layout; // Configuration to customize how the Meso experience is launched and presented

>   headlessSignature?: boolean; // Automatically invoke the `onSignMessageRequest` callback to initiate message signing

Reviewer: I wonder if we could say something here like:

```suggestion
  headlessSignature?: boolean; // Perform message signing in the background without prompting the user. This is useful for embedded wallets.
```

I'd like to convey the _why_ of this in documentation as well as the _what_

destroy(); // The meso iframe is unmounted. No more events/callbacks will fire.
```

### Headless Signature

> For partners using embedded wallets, message signing or the existence of a
> wallet itself may be transparent to the end user. In this scenario, it may not
> make sense for the user to be prompted to trigger message signing to either
> verify wallet ownership or approve a transaction.
> 
> To support this scenario, you can launch the Meso experience with the
> `headlessSignature` param set to `true`. In doing so, the `onSignMessageRequest`
> callback will be immediately invoked when needed, allowing you to return a
> signed message back to the Meso SDK using the embedded wallet and keeping this
> step completely transparent to the user. Once the signed message is received,
> the rest of the Meso experience proceeds as usual.

Reviewer: ```suggestion
In some cases (such as embedded wallets), message signing may be transparent to the end user. You can launch the Meso experience with `headlessSignature: true` which will invoke the `onSignMessageRequest` callback immediately allowing you to return a
signed message back to the Meso SDK using the embedded wallet and keeping this
step completely transparent to the user.
```

_purely a subjective suggestion_
